docs/upgrade-guides/2.0.md

```diff
@@ -331,7 +331,21 @@ Always pass a parameter.
 $menu = Timber::get_menu( 'primary' );
 ```
 
-@TODO
+#### The Pages Menu
+
+If you didn’t provide a parameter and didn’t have any menus registered, Timber would build **a menu from your existing pages**. This still works, but you have to use the new `Timber::get_pages_menu()` function.
+
+**Old**
+
+```php
+$menu = new Timber\Menu();
+```
+
+**New**
+
+```php
+$menu = Timber::get_pages_menu();
+```
```

Comment:
It's probably worth noting specifically that calling `Timber::get_menu()` without arguments will return `false` if there are no menus set up.

Reply:
Added in bb2fafe795b81826ff87d2b4bdc1512e567aeda3 👍